pipelines/templates/build-python-template.yaml

```diff
@@ -46,6 +46,12 @@ steps:
   displayName: 'Install python dependencies'
   inputs:
     targetType: 'inline'
+    script: |
```

Comment:
why are there two "script" elements to the task?

Reply:
good catch, I think that this occurred during one of the rebasings from master.  I created a commit to use the pipeline build from master with the additional step of downloading the large spacy model.